frontend/src/Editor/Editor.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<div class="buttons">
+		<Button
+			v-for="(module, i) in modules"
+			:key="module.title + i"
+			:title="module.description"
+			:icon="module.icon"
+			:action="module.action"
+		/>
+	</div>
+	<div id="editor" contenteditable spellcheck="false" @input="this.update">Enter text here...</div>
+</template>
+
+<script lang="ts">
+import { Options, Vue } from 'vue-class-component';
+import { debounce } from 'ts-debounce';
+import Button from '@src/Editor/Button.vue';
+import bold from './modules/bold';
+import italic from './modules/italic';
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import PatchMakeWorker from 'worker-loader!./patch-make.worker';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import PatchApplyWorker from 'worker-loader!./patch-apply.worker';
+
+// const socket = new WebSocket('ws://localhost:8080');
+// console.log(socket.readyState === WebSocket.CONNECTING);
+
+const patchMakeWorker = new PatchMakeWorker();
+patchMakeWorker.addEventListener('message', function (e) {
+	console.log('Patch by this user:');
+	console.log(e.data);
+});
+
+const patchApplyWorker = new PatchApplyWorker();
+patchApplyWorker.addEventListener('message', function (e) {
+	const editor = document.getElementById('editor');
+	if (editor) editor.innerHTML = e.data;
+	console.log('Patch by another user:');
+	console.log(e.data);
+});
+
+@Options({
+	components: {
+		Button,
+	},
+})
+export default class Editor extends Vue {
+	modules = [bold, italic];
+
+	execute(command: string, arg?: string): void {
```

Comment:
I think its ok to store buttons config inside this file in computed properties (we will get rid of long icon svg string in future). So this method is no longer needed (it is replaced by two onClick methods)